client/src/Extrinsic/components/ExtrinsicDetailsEventList.tsx

```diff
@@ -17,8 +17,8 @@ const ExtrinsicDetailsEventList: FC<Props> = ({ events }) => {
   const generateColumns = (events: Event[]): Column[] => [
     {
       title: "Event Id",
-      cells: events.map(({ block, pos }) => (
-        <div>{`${block.height}-${pos}`}</div>
+      cells: events.map(({ block, pos }, index) => (
+        <div>{`${block?.height || index}-${pos}`}</div>
```

Comment:
I believe `key` is missing here and in other places, there should be warnings in the console